src/siembol/Chart.yaml

```diff
@@ -0,0 +1,55 @@
+# Chart for siembol
+apiVersion: v2
+description: siembol is a security analytics framework akin to projects like Apache Metron
+name: siembol
+type: application
+version: 0.1.0
+#appVersion: 0.0.1
+keywords:
+- kafka
+- zookeeper
+- storm
+- gitlab
+- hdfs
+- security
+- metron
+home: https://github.com/G-Research/charts
+sources:
+- https://github.com/G-Research/siembol
+- https://github.com/G-Research/charts/siembol
+maintainers:
+- name: stackedsax
+  email: alex.scammon@ext.gresearch.co.uk
+- name: generalfuzz
+  email: headphonejames@gmail.com
+#icon: https://kafka.apache.org/images/logo.png
+
+dependencies:
+- name: storm
+  version: ~1.0.2
```

Comment:
can we use 1.2.1 version?

Reply:
I'm sure you know, but just making sure: 1.0.2 refers to the Chart version, not the version of Storm.  

However, the 1.0.2 chart is using the 1.1.1 version of Storm which isn't 1.2.1, the version you're looking for.  I don't think there's any reason for us not to update it to 1.2.1.  I'll attend to that tomorrow.

fwiw, you can see the tag here:

 * https://github.com/G-Research/charts/blob/master/src/storm/values.yaml

Reply:
Thanks for explanation. It shoudl eb fine when we will use storm 1.2.1 or 1.2.3

Reply:
I bumped the Storm chart version to 1.2.1.  Should be good to go now.

Reply:
This should all be updated with the 1.0.3 Storm chart that is pointing at the 1.2.1 Storm app version now.  Sound good for now?

Reply:
Perfect, thanks

Reply:
Actually, I think we'll bump this to 1.2.3, after all.  Seems that the docker image for Storm 1.2.1 is based on alpine where Storm 1.2.3 is debian.  For some strange reason, the alpine version was giving Jonathan fits, whereas the debian version was fine.

Reply:
Storm 1.2.3 should be fine. The code is compiled for 1.2.1 now but it should work in my opinion.

Reply:
Updated below to Storm 1.2.3 and I can confirm that the alerting topology now works fine.